otherlibs/stdune/src/proc.mli

```diff
@@ -1,3 +1,7 @@
+(** [Proc.restore_cwd_and_execve prog args ~env] runs [prog] with [args] in [env].
+
+  Note that [args] is not like [argv] in that [(Array.to_list args).(0)] is the first
+  argument rather than the conventional program name in [Unix.execve]. *)
```

Comment:
Maybe worth rewriting it in a way where it states that prog will be prepended to args to form `argv` as used in `Unix.execve`, because the way it is written it sounds a bit scary and requires knowledge of the C convention. Whereas it is actually a quality of life improvement and makes using it more robust.

Reply:
I've pushed a modification to the comment. Hopefully things are a bit clearer now.

Reply:
Yes, I think it is quite nice now. Thanks!